### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by Jack-bo1220** [antofuller/CROMA#2]
What a great job!
Looking forward to releasing the pretrained weight (CROMA :vit-large ) as soon as possible. I believe it will provide valuable assistance for geospatial community researchers.

Thanks.

**Comment by Jack-bo1220** [comment]
A confusion is bothering me, could the authors provide me with some help? Thanks a lot.

**Comment by antofuller** [comment]
Thanks for your interest! I'll upload the pretrained weights this weekend.

We use the same setup in Table 4 for BigEarthNet-MM as Table 1 for BigEarthNet. This means we fit a linear probe on 10% of the training samples and report results on the entire validation set.
